modularity/Modularity.ipynb

```diff
@@ -0,0 +1,360 @@
+{
```

Comment:
<p>Please coordinate with the team, these attributes should all be in the actual data. Also your group should be able to figure out what sex FAFB is</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neurodata/connectocross/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>